docs/api_docs/python/requirements.txt

```diff
@@ -12,5 +12,5 @@ sphinxcontrib-qthelp==1.0.3
 
 # IREE Python API
 -f https://iree.dev/pip-release-links.html
-iree-compiler
-iree-runtime
+iree-base-compiler
+iree-base-runtime
```

Comment:
This could continue using the old name until we publish a first release too. Move to https://github.com/iree-org/iree/pull/19065?

Reply:
Sure!